src/App.js

```diff
@@ -1,23 +1,22 @@
-import logo from './logo.svg';
 import './App.css';
+import { useContext } from 'react';
+import { GameContext } from './context/GameContext.js';
+import GameBoard from './components/GameBoard.js';
+import Reset from './components/Reset.js';
 
 function App() {
+  const { gameMessage } = useContext(GameContext);
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <main>
+        <div className="game-message">{gameMessage}</div>
+        <Reset></Reset>
```

Comment:
in React if you don't need to wrap a component around a child, you can just do self-closing ```<Reset />```